hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/resolver/TestMountTableResolver.java

```diff
@@ -552,6 +552,10 @@ public void testMountTableScalability() throws IOException {
 
     assertEquals(100000, mountTable.getMountPoints("/").size());
     assertEquals(100000, mountTable.getMounts("/").size());
+    for (int i = 0; i < 20; i++) {
```

Comment:
Add a comment saying this is for testing concurrency.

Reply:
Done, thank you!